chat/vulkan/amd64/Containerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi9/python-311:1-66
+USER 0
+RUN dnf install -y python3-dnf-plugin-versionlock
+RUN dnf install -y mesa-vulkan-drivers-23.3.3-1.el9.x86_64
+RUN dnf versionlock mesa-vulkan-drivers-23.3.3-1.el9.x86_64
+RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+RUN dnf install -y git cmake ninja-build gcc gcc-c++ 
+RUN dnf copr enable -y ligenix/enterprise-sandbox epel-9-x86_64
+RUN dnf install -y vulkan-headers vulkan-tools
```

Comment:
as it looks like it's not a multi-stage build, you should mergre layers and cleanup .rpm files / rpm cache at the end to reduce the size of the image

or maybe better, use the default image and copy the custom rootfs so we've a scratch/smaller image (this is how ubi micro is done)

Reply:
I've been trying to copy what they did on ubi-micro but i cannot make it work with installroot. There is something else to set up but i was not able to find it. 
I went to the good old way to check which folders to copy and i added the commands. If you know how to make the custom rootfs works, i'm all ears